src/pages/Home.vue

```diff
@@ -38,11 +45,69 @@ export default {
       default: "",
       type: String,
     },
+    program: {
```

Comment:
same suggestion as before of including it under purposeParams

Reply:
@sudheshna-donthineni I think we're gonna ditch all the other params right after this program thing is defined?

We talked about this on the call no? A little confused now because then how would we know the destination information? Oh maybe through the session thing pritam is working on?